src/trie/trie_type.c

```diff
@@ -88,7 +88,7 @@ Vector *Trie_Search(Trie *tree, char *s, size_t len, size_t num, int maxDist, in
     }
     // in prefix mode we also factor in the total length of the suffix
     if (prefixMode) {
-      ent->score /= sqrt(1 + fabs(slen - len));
```

Comment:
If I remember correctly, clang threw a warning for `fabs` here because `slen` and `len` are unsigned. But getting rid of `fabs` isn't the right solution, I should have cast each of `slen` and `len` to a signed type instead and kept the call to `fabs`. Nice catch. Will fix.